conf/broker.conf

```diff
@@ -1080,6 +1080,7 @@ loadBalancerAutoUnloadSplitBundlesEnabled=true
 loadBalancerNamespaceBundleMaxTopics=1000
 
 # maximum sessions (producers + consumers) in a bundle, otherwise bundle split will be triggered
+# (disable threshold check with value -1)
```

Comment:
@rdhabalia for the doc side, there are 2 `loadBalancerNamespaceBundleMaxSessions` in the [Pulsar configuration page](https://pulsar.apache.org/docs/en/next/reference-configuration/#broker), need to add the explanations for both?

![image](https://user-images.githubusercontent.com/50226895/144805266-fcc3fe7c-cd84-4766-b8fc-4600f803ce47.png)

Reply:
@rdhabalia - will you be able to update this documentation? If not, I still think we should merge this PR and maybe someone can follow up with a PR to update the docs?

Reply:
yes, these config are not related to this change and already exist, but will create a separate PR to address documentation concerns.

Reply:
Hi @rdhabalia do not forget to update docs, thanks.

Reply:
Hi @rdhabalia any updates on the docs?